src/file/file.route.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express';
+import { Router } from 'express';
+import { readFiles, uploadFile } from './file.controller';
+
+const fileRouter: Router = express.Router();
+
+fileRouter.get('/admin/:uuid', readFiles);
+fileRouter.post('/upload/:uuid', uploadFile);
```

Comment:
Please, move user id to query parameters for the request. Don't put them to the url.
Rename get endpoint. It's not related to admin, it's related to reading files first of all.